api-reference/_hidden/dxGanttHeaderFilterTexts/ok.md

```diff
@@ -5,7 +5,6 @@ default: 'Ok'
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies text for the "OK" button in the popup window.
```

Comment:
Does this property allows you to change the button's initial caption? That is, "OK"...

Reply:
Yes